packages/core/src/cart/cart-request-sender.spec.ts

```diff
@@ -11,6 +11,7 @@ import { getResponse } from '../common/http-request/responses.mock';
 import BuyNowCartRequestBody from './buy-now-cart-request-body';
 import Cart from './cart';
 import CartRequestSender from './cart-request-sender';
+import { CartSource } from './cart-source';
```

Comment:
would be better to use it from [packages/payment-integration-api/src/cart/cart-source.ts](https://github.com/bigcommerce/checkout-sdk-js/pull/1773/files#diff-173aaeb1e68e6b3807e1ce6e2da6072fc33a3b528581af0cbb1b5dd30dadc796)